src/algebra/ring/basic.lean

```diff
@@ -281,6 +281,12 @@ mt f.codomain_trivial_iff_map_one_eq_zero.mpr zero_ne_one
 lemma domain_nontrivial [nontrivial β] : nontrivial α :=
 ⟨⟨1, 0, mt (λ h, show f 1 = 0, by rw [h, map_zero]) f.map_one_ne_zero⟩⟩
 
+lemma is_unit_map (f : α →+* β) {a : α} (h : is_unit a) : is_unit (f a) :=
```

Comment:
Kevin made the same suggestion above. This is in the `ring_hom` namespace, with the intention that you write it `f.is_unit_map h` rather than `h.map f`. But I can change it if you prefer!

Reply:
What I mean is whether we can use `is_unit.map` to prove this theorem.

Reply:
Oh! Thanks, done.